### ISSUE TITLE
NVDA crashes everytime Microsoft Take a Test App Launches 

### ISSUE THREAD

**Issue by testertesting2789** [nvaccess/nvda#15354]
### Steps to reproduce:
1. Start NVDA (any year or version)
2. Press Windows button and type run then press Enter
3. Dialog will populate. Now Type in the following "ms-edu-secureassessment:https://wapt.cambiumtdsuat.com/student"
4. Press Ok. Window will populate with dialog, press Yes.
5. MSTAT app will populate and NVDA is running,
6. Observe after a few seconds NVDA crashes. User is unable to start a test.
7. Restarting NVDA again makes no difference as it keeps crashing

### Actual behavior:
NVDA crashes when running MSTAT app

### Expected behavior:
NVDA does not crash with MSTAT app

### NVDA logs, crash dumps and other attachments:
[nvda Log.zip](https://github.com/nvaccess/nvda/files/12488408/nvda.Log.zip)

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2023.1, 2021.3.3; reproducible on any
#### Windows version:
Windows 10 22H2 Build 19045.2728
Windows 11 22H2 Build  22521.2134
#### Name and version of other software in use when reproducing the issue:
Microsoft Take a Test App
#### Other information about your system:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes. Reproducible on all versions tried. 
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Brian1Gaff** [comment]
So, its an add on then, according to the report. Assumedly you don't know which, yet.
 Brian

-- 
***@***.*** 
Sent via blueyonder.(Virgin media) 
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.

**Comment by Brian1Gaff** [comment]
PS those logs are from old versions. I see no full log of what addons you have either. That site is also specific on using a private browser, an operating system higher than a certain version of windows 10 as well. I notice a link to a secure browser, is that what you have used,if so what is it?

I know for example that some versions of nvda and Waterfox classic do crash badly sometimes, but the latest beta/rc1 of nvda will allow the browser to  crash without going down itself. Brian

-- 
***@***.*** 
Sent via blueyonder.(Virgin media) 
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.

**Comment by testertesting2789** [comment]
@Brian1Gaff, the application, MsTAT is not an add on but is a EdgeChromium browser that is kind of a secure browser which gets updated from windows.  The application also comes updated with Windows Education and Pro Editions.  The link to access this type of Secure browser via run is "ms-edu-secureassessment:https://wapt.cambiumtdsuat.com/student"

Two new logs from Win 10 and 11 with NVDA 2023.2 with the issue being reproduced  have been attached 
[log nvda.zip](https://github.com/nvaccess/nvda/files/12526285/log.nvda.zip)

**Comment by Adriani90** [comment]
@testertesting2789 are you still having this issue with NVDA 2024.1 Beta 10?

**Comment by testertesting2789** [comment]
@Adriani90  yes, the issue still persists with NVDA 2024.1 Beta 10. The application crashes as soon as it starts.

**Comment by seanbudd** [comment]
To progress this, one should analyse the crash dump and log to get a feel of what is causing the crash.
This may be an issue with MS TAT however analysing the crash dump should provide more information.

**Comment by testertesting2789** [comment]
This issue no longer reproduces itself when testing for us. Able to interact with MSTAT tool when using NVDA, and the tools do not crash.

NVDA: Version 2024.1beta10.(2024.1.0.31202) & 2024.4rc1 (2024.4.0.34248)
MS TaT Version: Browser Secure V129
Win 11 Version: 23H2

**Comment by seanbudd** [comment]
Thanks for reporting back @testertesting2789
